pkgs/development/libraries/mesa/default.nix

```diff
@@ -215,6 +217,6 @@ stdenv.mkDerivation {
     changelog = "https://www.mesa3d.org/relnotes/${version}.html";
     license = licenses.mit; # X11 variant, in most files
     platforms = platforms.mesaPlatforms;
-    maintainers = with maintainers; [ vcunat ];
+    maintainers = with maintainers; [ primeos vcunat ]; # Help is welcome :)
```

Comment:
I noticed that we currently lack time for this package and I'd like to help, but I also lack time to update mesa after every stable release (see first review comment).